gnovm/tests/stdlibs/std/frame_testing.gno

```diff
@@ -8,5 +8,8 @@ func NewUserRealm(user Address) Realm {
 // NewCodeRealm creates a new realm for a published code realm with the given
 // pkgPath.
 func NewCodeRealm(pkgPath string) Realm {
+	if !isRealm(pkgPath){
```

Comment:
fmt

Reply:
I did a make fmt but I did not change the files

Reply:
Can you update `gnovm/Makefile`'s `fmt` rule with the following?

```
	go run ./cmd/gno fmt $(GNOFMT_FLAGS) ./stdlibs/... ./tests/files/extern/... ./tests/stdlibs/...
```

This directory was skipped.

Then run `make fmt` and we should be good to go.

Reply:
fixed on bb91f0175e328fbc3fe6c6dcdaafcfbb6efab5f3